hmac-examples/python/hmac-client.py

```diff
@@ -8,44 +8,47 @@
 from email.utils import formatdate
 import requests
 from time import mktime
-from urlparse import urlparse
+from urllib.parse import urlparse
 from pprint import pprint
 
 def get_token(uri, secret):
     httpdate = formatdate(timeval=mktime(datetime.now().timetuple()),localtime=False,usegmt=True)
     url_parts = urlparse(uri)
 
     string_to_sign = "{0}\n{1}".format(httpdate, url_parts.path)
-    print "string_to_sign: " + string_to_sign
-    hm = hmac.new(secret, string_to_sign,hashlib.sha256)
-    return "HMAC {0}".format(base64.b64encode(hm.digest())), httpdate
+    print("string_to_sign: " + string_to_sign)
+    hm_digest = hmac.new(secret.encode('utf-8'), string_to_sign.encode('utf-8'), hashlib.sha256).digest()
+    base64encodedDigest = base64.b64encode(hm_digest).decode('utf-8')
+    return "HMAC {0}".format(base64encodedDigest), httpdate
```

Comment:
```suggestion
    return f"HMAC {base64encodedDigest}", httpdate
```
using f-strings here might be more idiomatic for python 3? (although maybe this is just personal preference because I find them easier to read 😅)